roseau/load_flow/network.py

```diff
@@ -312,14 +312,15 @@ def lines_frame(self) -> gpd.GeoDataFrame:
                     line.bus1.id,
                     line.bus2.id,
                     line.parameters.id,
+                    line.length.m,
```

Comment:
Added the length to the dataframe. It is useful information and it also useful as part of the tooltip of the interactive map.